### PR TITLE
HR_64 Focus is not going to text field

### PR DESCRIPTION
**Pull request by veroglez**
Fixes: 
- https://issues.liferay.com/browse/LPS-167561
- https://issues.liferay.com/browse/LPS-168108

### CODE REVIEW

**Comment by eltonccdantas** [comment]
I'll take a look at it

**Comment by eltonccdantas** [comment]
Hey @veroglez ,  I've analyzed your pull request and the failures look unrelated to it, you may forward it if no other dev review is needed. 🎉 

[Comparison run](https://testray.liferay.com/home/-/testray/runs/compare?formDate=1682352831388&testrayRunIdA=2278405367&testrayRunIdB=2278543879)

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SearchControls.js [hunk 1]
```diff
@@ -27,8 +29,15 @@ const SearchControls = ({
...
 	searchValue,
-	setSearchMobile,
 }) => {
+	const searchInputRef = useRef(null);
+
+	useEffect(() => {
+		if (searchMobile && searchInputRef.current) {
+			searchInputRef.current.focus();
+		}
+	}, [searchMobile]);
```
Comment (markocikos) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SearchControls.js:39]: Do we need to set `null` as default for `searchInputRef`? It references an HTML element that we are unconditionally rendering. If no, check for `null` is also not needed.

**Review by markocikos — COMMENTED** [review]
@veroglez Looks good, just one question inline.

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SearchControls.js (hunk 1 above)
Comment (veroglez) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SearchControls.js]: @markocikos thanks for reviewing this!  it's not necessary to set the ref to `null`. I have update the commit (https://github.com/liferay-frontend/liferay-portal/pull/3263/commits/3ccb762004d504ce9f14d77d8dfa260c9885a9ee) removing `searchInputRef` from the condition.

**Review by markocikos — APPROVED** [review]
LGTM! :sparkles:

**Comment by markocikos** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**